benchmark/scripts/benchmark_dpo_loss.py

```diff
@@ -1,4 +1,5 @@
-from test.chunked_loss.test_dpo_loss import HF_DPO_Loss
```

Comment:
I believe this fn wouldn't work because right now return is a tuple
https://github.com/linkedin/Liger-Kernel/blob/aeb8ce38b637eeecd92058ad6bbc0fac88fb257f/benchmark/scripts/benchmark_dpo_loss.py#L120C1-L122C21

Reply:
also I believe other chunked loss benchmarking scripts would face the same issue

Reply:
Got it! Thanks for pointing that out~